Rego/EXOConfig.rego

```diff
@@ -260,8 +269,9 @@ tests[{
     "ReportDetails" : ReportDetailsString(Status, ErrorMessage),
     "RequirementMet" : Status
 }] {
-    ErrorMessage := "Wildcard domain (\"*\") in shared domains list, enabling sharing with all domains by default"
-    Status := count(SharingPolicyAllowedSharing) == 0
+    CalendarSharingPolicies := SharingPolicyCalendarAllowedAllDomains
```

Comment:
Shouldn't `CalendarSharingPolicies` be the ActualValue in line 268 instead of `input.sharing_policy`?